### DISCUSSION TITLE
Incorrect recording timestamp?

### DISCUSSION THREAD

**Discussion by ywu093** [blakeblackshear/frigate#3180] [Ask A Question]
I'm on 0.10.1 release running on Docker container, I got recording enabled for all the cams, however, I noticed the recording timestamp was 12hrs ahead of the actual time on the same day. 

Example: 
System timezone = New Zealand timezone (UTC +12)
Actual time = 09:00 or 9:00 am on 1 May
Recording timestamp (folder structure is 2022-05\01\21) = 21 (9:00 pm) on 1 May

It behaves the same on both 12 and 24 system time format.

I'm not sure if it was because of the system is in New Zealand timezone (UTC +12), but I assume it should support different timezone setting?

**Comment by NickM-27** [comment]
First thing would be to check that /etc/localtime is mapped correctly to the docker container

**Reply by ywu093** [reply]
Thanks!

I was wondering if I can just set the timezone variable TZ via the docker-compose.yml file when I first install the frigate image as below:

version: "3.9"
services:
  frigate:
     .............
    environment:
      FRIGATE_RTSP_PASSWORD: "password"
      TZ: Pacific/Auckland

**Reply by NickM-27** [reply]
Should be done like this:

```yaml
volumes:
      - /etc/localtime:/etc/localtime:ro
```

**Reply by trythatagain** [reply]
I experienced a similar problem when the clocks changed. I'd automatically added the TZ variable and had NickM-27 answer in my compose file. I can confirm you just need NickM-27 answer to make the time report correctly.

**Reply by ywu093** [reply]
Thanks both, I got the correct timezone by setting the TZ variable in the docker compose file, I guess the command above works too.
